app.py

```diff
@@ -33,12 +42,18 @@ def ui(debug, port):
 @click.option("--config-yaml", default=".env_config.yaml", help="configuration file used to configure service")
 @click.option("--single-run", default=False, help="it will loop infinitely pause at interval if set to true")
 @click.option("--tracing-enabled", default=False, help="allows you to expose tracing metrics using jaegar")
-# Initializing click function.
-def run(job_type, config_yaml, single_run, tracing_enabled):
-    """Perform machine learning model generation with input log data."""
+def run(job_type: str, config_yaml: str, single_run: bool, tracing_enabled: bool):
+    """ Perform machine learning model generation with input log data.
+
+    :param job_type: job type is used to provide user the ability to run one training or inference or both.
```

Comment:
take out "job type is used to" and simply say: 
:param job_type:  provide user the ability to run one training or inference or both.

Reply:
Fixed